### ISSUE TITLE
Embed JRE for Windows and Linux installation

### ISSUE THREAD

**Issue by mbastian** [gephi/gephi#2336]
We're successfully embedding the JRE on the Mac OS X application. This has worked very well, and avoided users to have to download Java separately (which is a pain, especially on Mac OS).

We should do the same for Windows and Linux packages. One way to do it might be to use [JLink](https://docs.oracle.com/javase/9/tools/jlink.htm).

### Definition of done
* Automated build Linux/Windows also embeds the JRE, and adapt the launcher configuration so it can be found
* The JRE version is consistent with Mac OS X

**Comment by wetneb** [comment]
I am interested in learning more about Gephi's packaging pipeline because I think OpenRefine has a lot to learn from it. In OpenRefine we use Launch4j for Windows packaging but I have no visibility about whether it is actually the right thing to use in 2021. It does let us embed a JRE though.

**Comment by mbastian** [comment]
This was implemented in the application's pom.xml, similar to the Mac OS way. We centralised the JRE download into it's own task to make it somewhat cleaner.

One major caveat is that we have to choose a specific architecture (x64) for the JRE. For users that have a different architecture this will not work and they will need to manually edit the `gephi.conf` file with an alternative JRE or comment it out to use the system's default Java. 

As a permanent fix we could either

1. Propose to download Gephi with or without the JRE
2. Propose more artifacts, one per architecture.

In both cases, it's extra work to build those packages so I would suggest to ship the x64 as default and write in the documentation how to fallback to the Java on your system.
